pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -51,17 +58,18 @@ func (r *resourceAllocationScorer) score(
 	if node == nil {
 		return 0, framework.NewStatus(framework.Error, "node not found")
 	}
-	if r.resourceToWeightMap == nil {
+	if r.weightList == nil {
```

Comment:
```suggestion
	if len(r.weightList) == 0 {
```

Reply:
I think this need return `0, nil`, otherwise th test case "Resources not set"(`resources:      nil`) will fail.

Or we just keep return a error `0, framework.NewStatus(framework.Error, "resources not found")`?